DTS-STN/Alpha-Site
# Thank you page

# Description

[success/thank you page after successful registration](https://trello.com/c/ESZzSCEo/241-241-success-thank-you-page-after-successful-registration)

Thank you after user has successfully submitted the form.
![image](https://user-images.githubusercontent.com/72703030/121743219-03756b80-cacf-11eb-9f02-c2d0b35a8a40.png)
![image](https://user-images.githubusercontent.com/72703030/121743449-6109b800-cacf-11eb-8f62-c10acc26739d.png)

## Acceptance Criteria

Fully translated and accessible

## Test Instructions

1. Fill the sign-up form correctly and submit, it should go to thank you page
2. Fill the sign-up form with mistakes and submit, it should not go to thank you page
3. Check the thank-you page on different screen sizes

## Checklist


## Product and Sprint Backlog

[Trello](https://trello.com/b/ZqWtJSyh/alpha-site-board)
